ftw/tokenauth/permissions.zcml

```diff
@@ -2,6 +2,7 @@
     xmlns="http://namespaces.zope.org/zope"
     i18n_domain="plone.restapi">
 
+  <permission id="ftw.tokenauth.ImpersonateUser" title="ftw.tokenauth: Impersonate user" />
```

Comment:
Could you also add this permission to `lawgiver.zcml`? 
I'd probably also just ignore it in lawgiver, so that integrators can configure it on the site root (=globally).
A `rolemap.xml` entry is not necessary because no roles have the permission by default, right?

Reply:
Amended.